packages/next/package.json

```diff
@@ -89,6 +89,7 @@
     ]
   },
   "dependencies": {
+    "@mswjs/interceptors": "0.23.0",
```

Comment:
Let's ncc this similar to our other dependencies and move this to `devDependencies`.

Reply:
done